package.json

```diff
@@ -1,16 +1,28 @@
 {
-  "name": "dependents-testing",
+  "name": "dependents",
   "version": "0.1.0",
```

Comment:
Should we increment the version in this PR too?

Reply:
Done